app/views/conversations/_conversation.haml

```diff
@@ -32,7 +32,7 @@
         .last_message
           - if conversation.messages.present?
             %em
-              = conversation.messages.last.text
+              = Diaspora::MessageRenderer.new(conversation.messages.last.text).markdownified
```

Comment:
`Conversation.find(2).messages.last.message.plain_text_without_markdown`.

To explain both of your mistakes:

* `conversation = Conversation.find(2) #=> <Conversation ...>` 
* `messages  = conversation.messages # => [<Message ....>, <Message ...>, ...]` (well, `AR::Relation` really, but conceptually.
* `message = messages.last # => <Message ...> `, note *not* `Diaspora::MessageRenderer`, `Message` does not respond to `plain_text_without_markdown`, only `Diaspora::MessageRenderer` does.
* `message.text #=> "foobar..."`
* `renderer = message.message # => <Diaspora::MessageRenderer ....>` same as `Diaspora::MessageRenderer.new(message.text)`
* Your second snippet effectively did `Diaspora::MessageRenderer.new(message)` and `Message#to_s` is `"<Message ....>"`.